src/Components/GalleryCarousel/GalleryCarousel.module.scss

```diff
@@ -0,0 +1,103 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+:root {
+  --controllerColor: rgba(196, 196, 196, 0.64);
+}
```

Comment:
Poppins is already imported in index.scss. Use that --srijan4-font1 variable in font-family

Also set --controller-color in index.scss' :root